README.md

```diff
@@ -1 +1,8 @@
-# munchkin_cardbase
\ No newline at end of file
+### To run dev envoirement follow instructions below: 
```

Comment:
then you have to write this) from your description people may think that this is an instruction on how to set up the development environment
with the documentation you need to be precise and very careful - you don't know who and when will use it and what experience he has